src/lib/localisation.ts

```diff
@@ -0,0 +1,60 @@
+/*
+	Date: April 2021
+	This is a reimplementation in DCR, for parity of the Localisation.scala file in frontend.
+	I realise that the replacement items are rather adhoc but again, we are just ensuring parity.
+*/
+
+// How to read the repacement instructions ?
+// ["US", "Film", "Movies"] , means: If the edition is US, then replace "Film" by "Movies"
+// Instructions in the caseInsensitive array must also work in lowecase, therefore replace "film" by "movies"
```

Comment:
Not a big deal, and 👍 for the useful explanation, but if you use the js-docs pattern here and put this text above the `export localise` line then it will appear in a tooltip in VSCode! See [useOnce](https://github.com/guardian/dotcom-rendering/blob/44bf079a459f5c29a5c3a03d946bbd28dd5df65a/src/web/lib/useOnce.ts#L3) for an  example